bundles/io/org.eclipse.smarthome.io.rest.test/src/test/groovy/org/eclipse/smarthome/io/rest/test/filter/CorsFilterTest.groovy

```diff
@@ -34,6 +34,8 @@ class CorsFilterTest {
     private static final String HTTP_GET_METHOD = "GET"
     private static final String HTTP_OPTIONS_METHOD = "OPTIONS"
 
+    private static final String CONTENT_TYPE_HEADER = "content-type";
```

Comment:
I understand your point completely, but this suggests that `containsHeaderWithValue` is not the correct test because it doesn't handle case-insensitive header field values:
```groovy
    private boolean containsHeaderWithValue(MultivaluedMap<String, String> headers, String header, String value) {
        return headers.getFirst(header) && (value ? headers.getFirst(header) == value : true)
    }
```
Perhaps what's needed is another method?
```groovy
    private boolean containsHeaderWithCaseInsensitiveValue(MultivaluedMap<String, String> headers, String header, String value) {
        return headers.getFirst(header) && (value ? headers.getFirst(header).equalsIgnoreCase(value) : true)
    }
```